### DISCUSSION TITLE
Rocket IP Add Seemed to Change with Update

### DISCUSSION THREAD

**Discussion by taylorwmj** [dani-garcia/vaultwarden#2485] [Q&A] [answered]
Noticed I was unable to get a login page at the host IP today upon after upgrading. Reviewing my systemd logs it appears Rocket is launched from localhost (127.0.0.1) instead of just 0.0.0.0 as it did previously.

OLD: `2022-05-23 12:55:04.563][start][INFO] Rocket has launched from http://0.0.0.0:8000`
NEW: `2022-05-23 15:40:07.131][start][INFO] Rocket has launched from http://127.0.0.1:8000`

I am able to access the web GUI by ssh-ing to my vaultwarden server, export my display, and running Firefox from a remote terminal and navigating to 127.0.0.1:8000. 

Suppose I need to update this IP Address....where can I do this?

**Comment by BlackDex** [comment, marked as answer]
You need to change the `ROCKET_ADDRESS` to be `0.0.0.0`
Which is the default, unless changed.

**Reply by taylorwmj** [reply]
Weird....makes sense, but don't know why I need it now. Have had this RS/Vaultwarden install for over a year now upgrading in the same way. I threw that into my `.env` and it did the trick.

**Reply by BlackDex** [reply]
It could be that you have ROCKET_PROFILE configured before now i think of it. But that has changed and could cause a different default address to be used.
